deconz/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 6.16.0
+
+- Bump deCONZ to 2.18.2
```

Comment:
maybe adding a new line here, to be consistent with the rest of the file

Reply:
@pfefferle Thank you, you're correct